src/ioctls/device.rs

```diff
@@ -110,30 +107,12 @@ mod tests {
     #[test]
     #[cfg(target_arch = "aarch64")]
     fn test_create_device() {
-        use kvm_bindings::kvm_device_type_KVM_DEV_TYPE_FSL_MPIC_20;
+        use ioctls::vm::create_gic_device;
 
         let kvm = Kvm::new().unwrap();
         let vm = kvm.create_vm().unwrap();
 
-        let mut gic_device = kvm_bindings::kvm_create_device {
-            type_: kvm_device_type_KVM_DEV_TYPE_FSL_MPIC_20,
-            fd: 0,
-            flags: KVM_CREATE_DEVICE_TEST,
-        };
-        // This fails on aarch64 as it does not use MPIC (MultiProcessor Interrupt Controller), it uses
-        // the VGIC.
```

Comment:
I think we can also keep this part as the error case? As this part is not really related to the `create_gic_device` function. @andreeaflorescu

Reply:
Yap, makes sense.